### PR TITLE
fix check mark on non supported keypairType

### PR DESCRIPTION
**Pull request by samaradel**
### Description

fix check mark on non-supported keypairType ```ed25519```

### Changes

image

### Related Issues

- [#2096](https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2096)

### Checklist

- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/weblets/profile_manager.vue [hunk 1]
```diff
@@ -157,7 +157,7 @@
                               :disabled="creatingAccount || activatingAccount || activating"
                               @click:append="reloadValidation"
                             >
-                              <template v-slot:prepend-inner v-if="mnemonic.length > 0">
+                              <template v-slot:prepend-inner v-if="mnemonic && keypairType !== KeypairType.ed25519">
```
Comment (AlaaElattar) [inline packages/playground/src/weblets/profile_manager.vue:160]: why u are not supporting this keypair type ??

Comment (AlaaElattar) [reply packages/playground/src/weblets/profile_manager.vue:160]: If i have mnemonic of type ed25519 and activated, I can login.

Comment (samaradel) [reply packages/playground/src/weblets/profile_manager.vue:160]: > why u are not supporting this keypair type ??

I see from the message that it's not supported

Comment (AlaaElattar) [reply packages/playground/src/weblets/profile_manager.vue:160]: It's not supported to create mnemonic or activate pre existing one, but u can login if u have one already.

Comment (AlaaElattar) [reply packages/playground/src/weblets/profile_manager.vue:160]: I have activated one of type ed25519 and i can connect with it.

Comment (samaradel) [reply packages/playground/src/weblets/profile_manager.vue]: fixed

**Review by zaelgohary — DISMISSED** [review]

**Comment by samaradel** [comment]
@zaelgohary It's the same validation of the input.

**Comment by samaradel** [comment]
> what about this comment ?? [#2096 (comment)](https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2096#issuecomment-1941677709)

fixed

**Comment by AlaaElattar** [comment]
- The icon appears while validating.
- It shouldn't appear until validation is done. what do u think ?
[Screencast from 02-19-2024 04:10:33 PM.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/7379e961-98cd-404e-aa27-63423127737c)

**Comment by samaradel** [comment]
it checks the mnemonic and length while input checks the same rules and if the mnemonic existed or not it's the same

**Comment by AlaaElattar** [comment]
> it checks the mnemonic and length while input checks the same rules and if the mnemonic existed or not it's the same

I think it shouldn't appear until all is valid

**Comment by maayarosama** [comment]
The icon still appears while validating
Peek 2024-02-20 12-39

**Review by AlaaElattar — APPROVED** [review]

**Review by MohamedElmdary — APPROVED** [review]

**Review by zaelgohary — APPROVED** [review]

**Pull request merged**